Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/audit/AdminDistTransforms.java

```diff
@@ -100,8 +100,7 @@ protected LogEventRequestType getLogEventRequestType(EDXLDistribution body, Asse
 
         result.setAuditMessage(auditMsg);
         result.setDirection(direction);
-        result.setInterface(_interface);
-
+        result.setRemoteHCID(HomeCommunityMap.formatHomeCommunityId(communityId));
```

Comment:
This method should be used for getting AD Remote homecommunityID - getAdminDistributionMessageCommunityID()

This method require few changes and needs to get hcid based on direction ignoring interface.